simple_sso/sso_server/models.py

```diff
@@ -52,6 +53,16 @@ def rotate_keys(self):
         self.save()
 
 
+def logout_token(sender, request, **kwargs):
```

Comment:
Same as with client... Why would this receiver be in a "models" file? It does not seem to be a model/data definition. Perhaps utilize a "signals.py" file